Makefile.am

```diff
@@ -107,9 +106,8 @@ additional_caps = CAP_DAC_OVERRIDE
 endif
 else
 ifp_dbus_exec_comment = \# "sss_signal" is used to force SSSD monitor to trigger "sssd_ifp" reconnection to dbus
-ifp_exec_cmd = $(sssdlibexecdir)/sss_signal
+ifp_dbus_exec_cmd = $(sssdlibexecdir)/sss_signal
 ifp_systemdservice =
-ifp_restart =
```

Comment:
This was removed intentionally?

Reply:
Yes, to make it consistent with other sssd-*.service files.